lib/logging/third_party_transaction.rb

```diff
@@ -27,10 +27,11 @@ def wrap_with_logging(*method_names, additional_class_logs: {}, additional_insta
           method_names.each do |method_name|
             # define patchable method(s) with the same name inside of a proxy module.
             define_method(method_name) do |*args, &block|
-              log_3pi_begin(method_name, additional_class_logs, additional_instance_logs, *args)
+              str_args = args.to_s
+              log_3pi_begin(method_name, additional_class_logs, additional_instance_logs, str_args)
```

Comment:
no need to pass full arguments when a string will do.  All we are doing is adding this stringified arg chain to our log.